components/shared_code/src/shared_data_model/sources/trello.py

```diff
@@ -43,32 +45,35 @@
     metrics=["issues"],
 )
 
+TRELLO_URL = "https://trello.com"
```

Comment:
Hmm, does the combining/extending of `HttpUrl` objects not work well?

Reply:
Yes, unfortunately `HttpUrl("https://example.org") / HttpUrl("path")` does not work.